cluster-autoscaler/cloudprovider/azure/README.md

```diff
@@ -197,6 +197,12 @@ The `AZURE_ENABLE_VMSS_FLEX` environment variable enables VMSS Flex support. By
 |---------------------------|---------|-----------------------------------------|---------------------------|
 | enableVmssFlex            | false   | AZURE_ENABLE_VMSS_FLEX                  | enableVmssFlex            |
 
+The `AZURE_GET_VMSS_SIZE_REFRESH_PERIOD` environment variable defines in seconds how frequently to call GET VMSS API to fetch VMSS info per nodegroup instance. By default, value is 30 seconds.
```

Comment:
This is currently only used for \[uniform\] spot, but there is nothing here that makes it clear (or in code, until the very last moment). At the very least, we should document this here and in comments. Even better would be to reflect this here, and in variable _names_ internally. (The only reason not to change environment and variable names would be if we intend to start using this for non-spot as well ...)